android/src/main/java/com/balthazargronon/RCTZeroconf/ZeroconfModule.java

```diff
@@ -134,6 +139,21 @@ public void onServiceResolved(NsdServiceInfo serviceInfo) {
             service.putString(KEY_SERVICE_HOST, serviceInfo.getHost().getHostName());
             service.putInt(KEY_SERVICE_PORT, serviceInfo.getPort());
 
+			WritableMap txtRecords = new WritableNativeMap();
+
+			Map<String, byte[]> attributes = serviceInfo.getAttributes();
+			for (String key : attributes.keySet()) {
+				try {
+					byte[] recordValue = attributes.get(key);
+					txtRecords.putString(String.format(Locale.US, "%s", key), String.format(Locale.US, "%s", recordValue != null ? new String(recordValue, "UTF_8") : ""));
```

Comment:
Should we use `Locale.getDefault()` instead?

Reply:
Good point